ONSdigital/ras-rm-survey
# Replace GORM with core database/sql

# Motivation and Context
GORM was hard to use and harder to unit test, so we're bailing back to using `database/sql`

# What has changed
Removed all ORM code
Implemented `golang-migrate` to handle DB schemas
Updated unit tests

# How to test?
`ras pf`
Run main.go in your IDE
Check your database has the surveyv2 schema with the setup below

![image](https://user-images.githubusercontent.com/6749550/98367347-5cf35480-202d-11eb-8ba2-46bb73bc2251.png)
